webview/src/plots/components/ribbon/styles.module.scss

```diff
@@ -73,12 +73,22 @@
 
 .list {
   display: flex;
+  position: sticky;
   align-items: center;
   gap: 6px;
   border-bottom: 1px solid $border-color;
   padding: 10px 15px;
   margin: 0;
   flex-wrap: wrap;
+  top: 0;
+  width: 100%;
+  z-index: 100;
+  background-color: var(--vscode-editor-background);
```

Comment:
variable `$bg-color` maps to `var(--vscode-editor-background)`. We try to only use scss variables (easier to use and understand).